src/main/kotlin/business/GameManager.kt

```diff
@@ -0,0 +1,26 @@
+package business
+
+class GameManager(val mines: Mines, openedCells: OpenedCells) {
+    private val _openedCells = openedCells
```

Comment:
```suggestion
class GameManager(val mines: Mines) {
    private val openedCells = openedCells
```

openedCells은 생성자가 아닌, 내부에 있어도 될 것 같아요. 게임이 진행되면서 내부적으로 openedCells 객체에 값을 추가할 수 있어 보여요.

Reply:
외부 노출을 하기 위해서 였습니다. 

내부로 작동으로 변경해보겠습니다.

Reply:
질문: 
- 필드 DI 가 trade off 가 있는 것 같은데  형준 님은 어떤 기준으로 DI를 사용하는지 궁금합니다.
  
DI 사용 장점:
- 유연성과 확장성
- 테스트 용이성
- 명시적 의존성
단점
- 불필요한 유연성
- 복잡성 증가

Reply:
저는 일단 필드 DI를 잘 사용하지는 않습니다. 장점과 단점은 흥광님이 적어주신 부분에 대해 매우 동의합니다.

사용할 때 건네주는 외부에서 건네주는 파라미터가 내부에서 쓰이며 가변적이나, 외부에 노출할 때는 불변을 지켜야할 때 흔히 사용을 하는 것 같습니다.

그 외에는 생성자로도 충분하다고 생각이 들어요. 🤔